src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -363,7 +363,10 @@ public static function create($uri, $method = 'GET', $parameters = array(), $coo
             case 'POST':
             case 'PUT':
             case 'DELETE':
-                if (!isset($server['CONTENT_TYPE'])) {
+                if (isset($server['content_type']) {
```

Comment:
Ok.. maybe fix it on the other side then? Ie. make sure the value is passed uppercase, as this comes from header/server conversion right?